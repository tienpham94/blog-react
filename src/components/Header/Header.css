.header {
  background: linear-gradient(to right, #24A3CF, #3650A0);
}

.container {
  display: flex;
  justify-content: space-between;
  max-width: var(--site-width);
  margin: 0 auto;
}

.title {
  color: white;
  font-size: 3.5rem;
  font-style: italic;
  text-transform: uppercase;
}

.list {
  display: flex;
  align-items: center;
  height: 100%;
}

.list-item {

}

.link {
  color: #fff;
  text-transform: uppercase;
  font-weight: bold;
  font-size: 2rem;
  margin-left: 2rem;
}

.link:focus,
.link:hover {
  color: #24A3CF;
}
