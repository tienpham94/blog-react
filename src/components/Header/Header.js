import React from 'react'
import { Link } from 'react-router'

import styles from './Header.css'

const Header = () => (
  <header className={styles.header}>
    <div className={styles.container}>
      <h1 className={styles.title}>Aletta</h1>
      <nav>
        <ul className={styles.list}>
          <li className={styles.listItem}><Link className={styles.link} to='/'>Posts</Link></li>
          <li className={styles.listItem}><Link className={styles.link} to='/about'>About</Link></li>
        </ul>
      </nav>
    </div>
  </header>
)

export default Header
