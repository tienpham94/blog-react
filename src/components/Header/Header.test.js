import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Header from './Header'

test('something', () => {
  const component = shallow(<Header />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
});
