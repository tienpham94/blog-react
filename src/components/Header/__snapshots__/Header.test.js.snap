exports[`test something 1`] = `
<header
  className="header">
  <div
    className="container">
    <h1
      className="title">
      Blog
    </h1>
    <nav>
      <ul
        className="list">
        <li
          className="listItem">
          <Link
            className="link"
            replace={false}
            to="/">
            Posts
          </Link>
        </li>
        <li
          className="listItem">
          <Link
            className="link"
            replace={false}
            to="/about">
            About
          </Link>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
